src/full_node/full_node.py

```diff
@@ -136,6 +136,7 @@ async def regular_start(self):
                 self.broadcast_uncompact_blocks(
                     self.config["send_uncompact_interval"],
                     self.config["target_uncompact_proofs"],
+                    self.config["sanitize_weight_proof_only"],
```

Comment:
This is not going to work for people, since we are no longer doing migrations of config. We need to make sure this works if the config variable is not set.

Reply:
ok I can default it to False I guess.